addons/event_sale/models/event.py

```diff
@@ -21,6 +21,7 @@ class event_event(models.Model):
     badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
+    is_unlimited = fields.Boolean('Unlimited Tickets', compute='_compute_seats_availability')
```

Comment:
Hello @tde-banana-odoo ,  It's for the ticket type(ticket) as limited and unlimited not for event. if i set seat_max 0 in ticket type then this ticket is unlimited not event,  there may another ticket type is limited for more See the Pad: https://pad.odoo.com/p/openerp-project.task-B4SM7EUAA9 (Point 8 (B>> 1,2,3,4,5 Point Explain by APR)) Thanks